src/main.c

```diff
@@ -0,0 +1,77 @@
+#include <stdlib.h>
+#include <libopencm3/stm32/rcc.h>
+#include <libopencm3/stm32/gpio.h>
+#include <libopencm3/cm3/systick.h>
+#include <libopencm3/stm32/iwdg.h>
+
+#include "cdcacm.h"
+#include "servo.h"
+#include "led.h"
+#include "systick.h"
+#include "global_vars.h"
+#include "i2c.h"
+
+void init(void);
+void jump_to_bootloader(void);
+
+int main(void) {
+    init();
+
+    // Signal we initialised
+    set_led(LED_STATUS_BLUE);
```

Comment:
Please could you add the meaning of the LEDs to the README. We should update the runbook too once we've reflashed.